examples/multi_nic_common/main.tf

```diff
@@ -56,8 +53,8 @@ module "bootstrap" {
   service_account = module.iam_service_account[each.value.service_account_key].email
   location        = each.value.location
   files = merge(
-    { for k, v in var.vmseries : "files/${k}/config/bootstrap.xml" => "${k}/config/bootstrap.xml" },
-    { for k, v in var.vmseries : "files/${k}/config/init-cfg.txt" => "${k}/config/init-cfg.txt" },
+    { for k, v in var.vmseries : "files/${k}/config/bootstrap.xml" => "${k}/config/bootstrap.xml" if can(v.bootstrap_template_map) },
```

Comment:
How do we handle `authcodes` in the example? How we do the licensing in general?

Reply:
Currently authcodes are not handled in the examples.